specification/recoveryservices/resource-manager/readme.md

```diff
@@ -41,7 +41,17 @@ model-validator: true
 semantic-validator: true
 message-format: json
 ```
+### Tag: package-2021-06
+
+These settings apply only when `--tag=package-2021-06` is specified on the command line.
 
+```yaml $(tag) == 'package-2021-06'
+input-file:
+  - Microsoft.RecoveryServices/stable/2021-07-01/registeredidentities.json
+  - Microsoft.RecoveryServices/stable/2021-07-01/replicationusages.json
+  - Microsoft.RecoveryServices/stable/2021-07-01/vaults.json
+  - Microsoft.RecoveryServices/stable/2021-07-01/vaultusages.json
+```
```

Comment:
The default tag is package-2021-07 but here we add a tag package-2021-06

Reply:
@deymadhumanti : please take care of this before merge

Reply:
Updated the version